shared/models/videos/import/video-import-live-status.enum.ts

```diff
@@ -0,0 +1,6 @@
+export const enum VideoImportLiveStatus {
```

Comment:
If this enum is not shared with the client, put it directly in the youtube-dl module

Reply:
Renamed and moved.